apps/gro/steps.js

```diff
@@ -133,7 +133,96 @@ module.exports = {
     template: 'email',
     fields: ['email-text'],
     continueOnEdit: true,
-    next: '/post'
+    next: '/country'
+  },
+  '/country': {
+    fields: [
+      'country'
+    ],
+    forks: [{
+      target: '/address-outside',
+      condition: function checkIfUK(req) {
+        return req.form.values.country !== 'United Kingdom';
+      }
```

Comment:
another example of where expanding `next` would be a benefit as this gives weight to one option